samples/nrf9160/lte_ble_gateway/README.rst

```diff
@@ -81,6 +81,15 @@ Building and running
 
 .. include:: /includes/build_and_run_ns.txt
 
+.. note::
+   To build the sample successfully, you must specify the board version along with the build target.
+   The board version is printed on the label of your DK, just below the PCA number.
+   For example, for board version 1.1.0, the sample must be built in the following way:
+
+   .. parsed-literal::
+      :class: highlight
+
+      west build --board nrf9160dk_nrf9160_ns@1.1.0
```

Comment:
I think it should be specified that this applies for building the hci_uart app for the nrf52840 board controller as well, not just for the nRF9160.

Reply:
I have moved the note to "Programming the sample" and now have 2 notes to make it clearer - One for nrf52840 and other for nRF9160.
Also updated the known issue section. Please review.